src/bundle/Controller/ContentController.php

```diff
@@ -381,4 +383,78 @@ public function updateMainTranslationAction(Request $request): Response
 
         return $this->redirectToRoute('ezplatform.dashboard');
     }
+
+    /**
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function hideAction(Request $request): Response
+    {
+        $form = $this->createForm(ContentType::class);
+        $form->handleRequest($request);
+
+        if (!$form->isSubmitted() || !$form->isValid()) {
+            throw new BadRequestHttpException();
```

Comment:
I think we used to return information with errors

Reply:
Because we can not find place where similar notification are displayed do not take this into account